model/cli/spans.yaml

```diff
@@ -0,0 +1,21 @@
+groups:
+  - id: span.cli
```

Comment:
I need help with this for pleasing policies-check

Reply:
it  should be `span.cli.internal`

```suggestion
  - id: span.cli.internal
```